api/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -155,7 +155,7 @@ export class SyncChefsDataService {
           MAX_PROJECT_TITLE_LENGTH
         ),
         totalEstimatedCost: responseData.submission.data.totalEstimatedCostOfProject,
-        asks: responseData.submission.data.totalRequestBeingMadeOfBcaapACDNotToExceedB,
+        asks: responseData.submission.data.totalRequestBeingMadeOfBcatACDNotToExceedB,
```

Comment:
We don't have this field in either of the forms: `submission.data.`**totalRequestBeingMadeOfBcatACDNotToExceedB**

Reply:
I don't quite know what we'll use/ what we'll remove and I'm not sure what the response body from CHEFS looks. like at the moment

Reply:
what's it called in the response, can update it to that